tests/common/include/tests/common/tools/ray_scan_utils.hpp

```diff
@@ -170,11 +170,6 @@ inline auto trace_intersections(const record_container &intersection_records,
         inline auto &volume_id() const { return entry.first; }
         inline auto &volume_link() const { return entry.second.volume_link; }
         inline auto &dist() const { return entry.second.path; }
-        inline auto r() const {
```

Comment:
Similar to the navigation inspector, in the intersection tracer, I print the global position of the intersections to see in which region of the detector a surface was missed. Could we somehow reuse these functions to print that information?

The output looks like this:
```
...
-------Intersection trace
ray gun: 	vol id: 35, dist:1498.95 [glob: r:108.776, z:1495 | loc: 108.776, -2.38761], (sf index:vol = 35 | id = 0 | index = 182 | extra = 255, links to vol:34), status: inside, direction: along
navig.: dist:0 [glob: r:94.9516, z:1305 | loc: 94.9516, -2.38761], (sf index:vol = 34 | id = 0 | index = 178 | extra = 255, links to vol:33), status: inside, direction: along
-------Intersection trace
...
```
When the tool tells me which entry in the trace differs, I can go back to this trace to see where the hole in the track happened. Local coordinates alone are not so helpful here ...